packages/common/src/core/types/types.ts

```diff
@@ -17,3 +17,5 @@ export const EntityResourceMap: {
 } = {
   location: "location.json",
 };
+
+export type IHubLocationType = "none" | "custom" | "default" | "org-extent";
```

Comment:
so for content our design was `"none" | "item" | "automatic"` and I think automatic was the convex hull of the features (if available) falling back to the item's extent. We need to coordinate about what are the right values that work for both content and and other entities. cc @thomas-hervey

Reply:
My apologies, re-looked over the confluence docs and realized where I missed that. I'll be adding `"item" | "automatic"` on in.

Reply:
After talking over with Tom we are on the same page I believe, and this type has been updated.